python/GafferSceneUI/InstancerUI.py

```diff
@@ -483,6 +483,22 @@ def __init__( self, headings, toolTipOverride = "" ) :
 
 		],
 
+		"omitDuplicateIds" : [
+
+			"description",
+			"""
+			When false, having the same ids on multiple points is considered
+			an error. Setting to True will allow a render to proceed, with any
+			points that share an id all removed.
+			""",
```

Comment:
Capitalisation is mismatched between `false` and `True` - maybe on and off are better anyway? 

> with any points that share an id all removed.

Sounds a bit awkward - maybe "with all instances that share an id being omitted?"